### DISCUSSION TITLE
[Bug] esrun 依赖的 @swc/core 需要升级，否则会和 @swc-node/core@1.13.3 不兼容

### DISCUSSION THREAD

**Discussion by XGHeaven** [midwayjs/midway#3954] [Q&A]
### Describe the problem（描述问题）

简单来说，虽然 `@swc-node/register` 和 `@swc/core` 锁死了版本，但是 `@swc-node/register` 内部没有锁死 `@swc-node/core` 的版本，为 "^1.9.1"，导致安装了最近刚发布的 `1.13.3`，而其内部默认启用了 `keepImportAttributes` 这个参数。

esrun 内部锁死的 `@swc/core@1.3.24` 并不支持这个参数，导致新安装的项目运行报错。
看了下 `@swc-node/core` 的 peer 是需要 `@swc/core >= 1.4.13` 的。

### Midway Versions（Midway 版本）

✓ @midwayjs/faas-typings(3.10.13)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(3.14.0)
✓ @midwayjs/async-hooks-context-manager(3.16.2)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(3.16.2)
✓ @midwayjs/bull(not installed)
✓ @midwayjs/bull-board(not installed)
✓ @midwayjs/cache-manager(not installed)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.16.2)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(not installed)
✓ @midwayjs/decorator(3.16.2)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(3.16.5)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(not installed)
✓ @midwayjs/info(not installed)
✓ @midwayjs/jwt(not installed)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.16.5)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/mqtt(not installed)
✓ @midwayjs/oss(not installed)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(not installed)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(not installed)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(not installed)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/tenant(not installed)
✓ @midwayjs/typegoose(not installed)
✓ @midwayjs/typeorm(not installed)
✓ @midwayjs/upload(not installed)
✓ @midwayjs/validate(not installed)
✓ @midwayjs/version(not installed)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(not installed)
✓ @midwayjs/ws(not installed)

**Comment by czy88840616** [comment]
用 lock 文件锁死版本。
